### PR TITLE
Issue #2070 - Support _total with _include and _revinclude

### PR DESCRIPTION
**Pull request by tbieste**
Issue #2070 - Support _total with _include and _revinclude

Signed-off-by: Troy Biesterfeld <tbieste@us.ibm.com>

### CODE REVIEW

**Review by prb112 — APPROVED** [review]
nice - I didn't realize it'd be that direct of an enablement

File: fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java [hunk 1]
```diff
@@ -723,6 +726,7 @@ public void testSearchIncludeIteratePrimaryType() {
         WebTarget target = getWebTarget();
         Response response =
                 target.path("Patient")
+                .queryParam("_total", "none")
```
Comment (michaelwschroeder) [inline fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java:729]: could add an assertNull(bundle.getTotal()) in this testcase

File: fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java [hunk 2]
```diff
@@ -891,6 +894,7 @@ public void testSearchRevIncludeIteratePrimaryType() {
         WebTarget target = getWebTarget();
         Response response =
                 target.path("Patient")
+                .queryParam("_total", "none")
```
Comment (michaelwschroeder) [inline fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java:897]: same comment as above - could add an assertNull(bundle.getTotal()) in this testcase

**Review by michaelwschroeder — APPROVED** [review]
LGTM - just a couple minor comments

File: fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java (hunk 1 above)
Comment (tbieste) [reply fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java:729]: Added.

File: fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java (hunk 2 above)
Comment (tbieste) [reply fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java:897]: Added.

**Comment by tbieste** [comment]
Comments addressed. Ready to merge when builds succeed.

**Pull request merged**